addons/website/views/snippets/s_faq_horizontal.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_faq_horizontal" name="Topics">
+    <section class="s_faq_horizontal" data-oe-shape-data="{'shape':'web_editor/Origins/09_001','flip':[],'showOnMobile':false,'shapeAnimationSpeed':'0'}">
+        <div class="o_we_shape o_web_editor_Origins_09_001"/>
+        <div class="container">
+            <div class="row">
```

Comment:
Defining multiple rows will limit the use of the grid feature only on the first row, so only on your heading.
Was it on purpose ? If not, I would prevent the use of grid for this snippet 🙂 .